Formula/l/localai.rb

```diff
@@ -0,0 +1,50 @@
+class Localai < Formula
+  include Language::Python::Virtualenv
+
+  desc ":robot: The free, Open Source OpenAI alternative"
```

Comment:
Since free and open source are required for Homebrew/core, this seems like a better description
```suggestion
  desc "OpenAI alternative"
```

Reply:
yeah, let's do `OpenAI alternative` :)